plugins/transforms/tokenreplacement/src/main/doc/tokenreplacement.adoc

```diff
@@ -0,0 +1,19 @@
+:documentationPath: /plugins/transforms/
+:language: en_US
+:page-alternativeEditUrl: https://github.com/apache/incubator-hop/edit/master/plugins/transforms/abort/src/main/doc/abort.adoc
+= Abort
+
+== Description
+
+This tranform type allows you abort a pipeline upon seeing input. It's main use is in error handling. For example, you can use this transform so that a pipeline can be aborted after x number of rows flow to over an error hop.
+
+== Options
+
+[width="90%", options="header"]
+|===
+|Option|Description
+|Transform name|Name of the transform.
+|Abort threshold|The threshold of number of rows after which to abort the pipeline. E.g. If threshold is 0, the abort transform will abort after seeing the first row. If threshold is 5, the abort transform will abort after seeing the sixth row.
+|Abort message|The message to put in the log upon aborting. If not filled in a default message will be used.
+|Always log|Always log the rows processed by the Abort transform. This allows the rows to be logged although the log level of the pipeline would normally not do it. This way you can always see in the log which rows caused the pipeline to abort.
+|===
```

Comment:
Just change the title and clean out the file and create a ticket if you are not writing the docs, then we have a placeholder and an action

Reply:
FIle has been cleaned up and ticket has been opened.